src/app/group/tabs/result-tab/result-tab.component.ts

```diff
@@ -27,20 +27,20 @@ export class ResultTabComponent implements OnInit {
   constructor(private userService: UserService, private route: ActivatedRoute) { }
 
   ngOnInit() {
-  this.currentGroupId = this.route.snapshot.paramMap.get('groupId');
-
-  this.userService.getStudentsByGroupId(this.currentGroupId)
-    .subscribe(students => { this.students = students; this.fillModel(students) });
-
-
   this.userService.getCuratorByGroupId(this.currentGroupId)
     .subscribe(curator => this.curator = curator);
 
-
   this.userService.getUserById(this.currentUserId)
   .subscribe(user => this.currentUser = user);
+  }
 
-}
+  ngOnChanges() {
+    this.currentGroupId = this.route.snapshot.paramMap.get('groupId');
+
+    this.userService.getStudentsByGroupId(this.currentGroupId)
+    .subscribe(students => { this.students = students; this.fillModel(students) });
+    console.log(this.students);
```

Comment:
should we keep console.log ? and don't forget about spaces ;)

Reply:
fixed